sdk/core/azure-core/src/http/curl/curl.cpp

```diff
@@ -25,17 +25,7 @@
 #include <thread>
 
 namespace {
-// Can be used from anywhere a little simpler
-inline void LogThis(std::string const& msg)
-{
-  if (Azure::Core::Logging::Details::ShouldWrite(
-          Azure::Core::Http::LogClassification::HttpTransportAdapter))
-  {
-    Azure::Core::Logging::Details::Write(
-        Azure::Core::Http::LogClassification::HttpTransportAdapter,
-        "[CURL Transport Adapter]: " + msg);
-  }
-}
+std::string const LogMsgPrefix = "[CURL Transport Adapter]: ";
```

Comment:
This seems fine for now.  I suggest we standardize log message formats in the design guidelines

Reply:
I like the `string serviceId`. But let's discuss.

Reply:
curl isn't a service. How about `componentId`?

Reply:
I agree